src/InputWithLabel.js

```diff
@@ -0,0 +1,26 @@
+
+import React from "react";
+   const InputWithLabel = ({
+      value,
+      handleTitleChange,
+      isFocused,
+      children
+   }) =>{
+      
+   const inputRef = React.useRef();
+   
+   React.useEffect(() => {
```

Comment:
-  Since the useEffect's dependency is undefined the hook won't run. It would be better to not pass a dependency array at all so it will run after every single render (and re-render) of your component. Something like this: 

`useEffect(() => {
    inputRef.current.focus()
  })`